### DISCUSSION TITLE
有没有大神能写个 wolframalpha 的计算插件

### DISCUSSION THREAD

**Discussion by zsychn** [lobehub/lobehub#1836] [General]
这个如果接上了，能解决太多涉及到逻辑计算的问题了，有没有大神能写一个，不会写代码研究2天发现搞不太定。

**Comment by arvinxx** [comment]
我 Q2 有空做一个，应该不麻烦

**Reply by arvinxx** [reply]
记到 https://github.com/lobehub/lobe-chat/discussions/1310 这里了。

**Reply by zsychn** [reply]
哇，太棒了！
